templates/app.yaml

```diff
@@ -406,6 +406,12 @@ Resources:
                 - ec2.amazonaws.com
             Action:
               - 'sts:AssumeRole'
+          - Effect: Allow
+            Principal:
+              Service:
+                - ec2.amazonaws.com
+            Action:
+              - 'sts:TagSession'
```

Comment:
you don't need another list item. Just remove this item and move - 'sts:TagSession' under - 'sts:AssumeRole'